toml/src/test/kotlin/org/rust/toml/completion/CargoTomlCompletionTestBase.kt

```diff
@@ -8,14 +8,17 @@ package org.rust.toml.completion
 import org.intellij.lang.annotations.Language
 import org.rust.FileTree
 import org.rust.RsTestBase
+import org.rust.ide.experiments.RsExperiments
 import org.rust.lang.core.completion.RsCompletionTestFixture
+import org.rust.openapiext.setFeatureEnabled
 
 abstract class CargoTomlCompletionTestBase : RsTestBase() {
 
     protected lateinit var completionFixture: RsCompletionTestFixture
 
     override fun setUp() {
         super.setUp()
+        setFeatureEnabled(RsExperiments.CRATES_LOCAL_INDEX, false)
```

Comment:
BTW if we actually need this, we need to restore the default value on test finish